src/objects/zcl_abapgit_object_msag.clas.abap

```diff
@@ -365,4 +424,70 @@ CLASS ZCL_ABAPGIT_OBJECT_MSAG IMPLEMENTATION.
     serialize_texts( io_xml ).
 
   ENDMETHOD.
+
+  METHOD delete_msgid.
+
+    delete_documentation( iv_message_id ).
+
+    DELETE FROM t100a WHERE arbgb = iv_message_id.
+    IF sy-subrc = 0 OR sy-subrc = 4.
```

Comment:
hmm? in case of success or error then perform the code in the IF?

Reply:
That's what the original SAP code does...
![image](https://user-images.githubusercontent.com/2453277/45716045-ae120700-bb8d-11e8-80ab-ee1c8442131c.png)

PS: I noticed now that resolve doesn't add the comment...